lib/web_ui/test/golden_tests/engine/compositing_golden_test.dart

```diff
@@ -405,6 +405,11 @@ void _testCullRectComputation() {
   // enough to fit the rotated clip.
   test('clips correctly when using 3d transforms', () async {
     final SurfaceSceneBuilder builder = SurfaceSceneBuilder();
+
+    builder.pushTransform(Matrix4.diagonal3Values(
+        EngineWindow.browserDevicePixelRatio,
+        EngineWindow.browserDevicePixelRatio, 1.0).storage);
```

Comment:
This change wouldn't be necessary if we applied 1/DPR to `<flt-scene>` when the root transform is missing.

I think some of our benchmarks will fail for the same reason.

Reply:
Agreed. Would like to change ViewConfiguration so we don't have to do this in the first place on the framework side.